packages/react-storage/src/components/StorageBrowser/Views/Controls/Message.tsx

```diff
@@ -88,20 +94,22 @@ interface MessageControlProps {
 interface _MessageControl<
   T extends StorageBrowserElements = StorageBrowserElements,
 > extends Pick<T, 'Icon' | 'View'> {
-  ({ variant }: MessageControlProps): React.JSX.Element;
+  ({ variant }: MessageControlProps): React.JSX.Element | null;
```

Comment:
Is this weird? If you dismiss the message it should be removed from the DOM

Reply:
Think it makes sense for at least the immediate term